conf/swisnap-init.sh

```diff
@@ -27,29 +27,29 @@ swisnap_config_setup() {
         exit 1
     fi
 
-    yq w -i "${PUBLISHER_APPOPTICS_CONFIG}" v1.publisher.publisher-appoptics.all.token -- "${SWI_TOKEN}"
-    yq w -i "${PUBLISHER_APPOPTICS_CONFIG}" v2.publisher.publisher-appoptics.all.endpoint.token -- "${SWI_TOKEN}"
-    yq w -i "${PUBLISHER_PROCESSES_CONFIG}" v2.publisher.publisher-processes.all.endpoint.token -- "${SWI_TOKEN}"
+    yq eval -i '.v1.publisher."publisher-appoptics".all.token = "'"${SWI_TOKEN}"'"' "${PUBLISHER_APPOPTICS_CONFIG}"
+    yq eval -i '.v2.publisher."publisher-appoptics".all.endpoint.token = "'"${SWI_TOKEN}"'"' "${PUBLISHER_APPOPTICS_CONFIG}"    
+    yq eval -i '.v2.publisher."publisher-processes".all.endpoint.token = "'"${SWI_TOKEN}"'"' "${PUBLISHER_PROCESSES_CONFIG}"   
```

Comment:
we could see this is updating the processor token

![processor-v2](https://github.com/user-attachments/assets/3607fbd8-905a-4252-93a8-3d6ccdd4ecee)